spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -189,8 +196,10 @@ def controller
       let(:view_spec) do
         Class.new do
           include ViewExampleGroup::ExampleMethods
+          def _default_file_to_render; end
```

Comment:
What needs this?

Reply:
[This](https://github.com/rspec/rspec-rails/pull/2264/files/87aee32e014aed4c818156a9c473c5de8682b8cc#diff-0cdf276129b0b83c6314283584c4df6eR205) and [this](https://github.com/rspec/rspec-rails/pull/2264/files/87aee32e014aed4c818156a9c473c5de8682b8cc#diff-0cdf276129b0b83c6314283584c4df6eR214).

Reply:
Aside, can you quote code as well as link, Github makes it really hard to navigate with just links...

Reply:
Ok I see it now, it's just a bit weird because its not in the same context.